kernel/kernel-api/src/main/java/io/delta/kernel/internal/replay/ActionsIterator.java

```diff
@@ -157,19 +175,55 @@ private void tryEnsureNextActionsIterIsReady() {
         }
     }
 
+    /**
+     * Get an iterator of actions from a checkpoint manifest that may contain sidecar files. If the
+     * current read schema includes Add/Remove files, then inject the sidecar column into this
+     * schema to read the sidecar files from the checkpoint manifest. When the returned
+     * ColumnarBatches are processed, these sidecars will be appended to the end of the file list
+     * and read as part of a subsequent batch (avoiding reading the checkpoint manifest more than
```

Comment:
This file is not called "checkpoint manifest". Please use "v2 checkpoint" / "v2 checkpoint file" / "v2 checkpoint top-level file" depending on the context everywhere in the PR.

Reply:
Should be fixed.